build.gradle.kts

```diff
@@ -15,12 +15,12 @@ allprojects {
       if (name != rootProject.name) {
         kotlin {
           ktfmt(libs.versions.ktfmt.get()).googleStyle()
-          targetExclude("**/generated/**")
+          target("**/*.kt")
```

Comment:
that's so weird that you have to specify this explicitly given it's a kotlin formatter 🤷

Reply:
Agreed. Usually the way it works is that Spotless automatically infers the source sets but for Android builds it doesn’t do that. There’s a warning about it here: https://github.com/diffplug/spotless/tree/main/plugin-gradle#kotlin